woocommerce/woocommerce-android
# Introduce Attributes Created view when creating first Variation

Summary
==========
Fixes issue #4209. As part of the Variation changes, the design team has decided to include an intermediary screen before creating the first variation to explain to the user what is happening in a better fashion.

Screenshots
==========
| Expected  | Actual |
| ------------- | ------------- |
| <img width="280" height="585" alt="image" src="https://user-images.githubusercontent.com/5920403/124674740-ad0f0900-de91-11eb-9f7b-822f49547159.png"> | <img src="https://user-images.githubusercontent.com/5920403/124765976-dfaa1780-df0c-11eb-82c6-bd7f361fdba1.png" width="280" height="585" /> |

Showcase
==========
<img src="https://user-images.githubusercontent.com/5920403/124764836-c2c11480-df0b-11eb-8781-49e8906c7800.gif" width="280" height="585" />


How to Test
==========
- Create a new Variable Product
- Create your first variation.
- See that before creating the variation there is an intermediary screen telling you that you are ready to create your first variation!
- Create your first variation and go to the Variation section to check if everything got created correctly

Update release notes:
